### DISCUSSION TITLE
Enum support

### DISCUSSION THREAD

**Discussion by return215** [supabase-community/supabase-csharp#129] [Q&A]
I haven't seen anything about using enum (Database Enumerated Types) anywhere in the docs. Will there be support for it?

I have looked at [Postgres docs](https://www.postgresql.org/docs/current/datatype-enum.html) and it seems like they just use the string value of the enum duirng operations.

**Comment by acupofjose** [comment]
There's not actually anything written in the docs for them (something that I should get around to doing) but enums are supported to a degree. There's a PR [here](https://github.com/supabase-community/postgrest-csharp/pull/72) outlining discussion around enums. But essentially, enums should be exact matches between the database and the C# code, casing, declaration order, etc. If there's specific functionality around enums that's missing, feel free to open an issue and/or a PR!
